### ISSUE TITLE
all training pair are positive?

### ISSUE THREAD

**Issue by hedes1992** [bertinetto/siamese-fc#49]
Recently I am convert the matconvnet training code to pytorch version, but meet some questions.
I find that in "training/vid_get_random_batch.m", the labels here are all 1 because here just considering positive pairs. But in "training/experiment.m", in the function get_batch.m, the label_inputs will always be like a gaussian shape(because the center is 1, and the other is 0 etc). So this might train the network output a gaussian shape score map all the time.
I train it in my pytorch code, also find the loss is constant in convergence.
![image](https://user-images.githubusercontent.com/14728901/37820181-d78291e6-2eba-11e8-9464-fdfa0089702c.png)

I guess this might because the label'shape is constant and the output is trained to constant
